src/Ng/NgJobFactory.cs

```diff
@@ -14,14 +14,14 @@ public static class NgJobFactory
         public static IDictionary<string, Type> JobMap = new Dictionary<string, Type>()
         {
             { "package2catalog", typeof(Package2CatalogJob) },
-            { "feed2catalog", typeof(Feed2CatalogJob) },
+            { "db2catalog", typeof(DB2CatalogJob) },
```

Comment:
Should I change Db2AzureSearch to DB2AzureSearch?

nit: tbh I've seen "Db" in our code more than "DB". For example `DbSet`, `GalleryDb` config section name, `TestDbAsyncQueryProvider` for mocking. Because of this, I think we should prefer `Db2Catalog` + `Db2Lucene` instead of `DB2Catalog` + `DB2Lucene`.

Reply:
`DB2AzureSearch` is the correct pascalcase but it's frequently mistaken. I don't think it's a big deal--if it's a ton of work to switch to it, I wouldn't worry about it, but if it's at the stage that you could quickly change to it then I think you should.

[Basically, abbreviations with only 2 letters are both capitalized with pascalcase.](https://stackoverflow.com/questions/15526107/acronyms-in-camelcase) I remember I used to get this wrong as well until @shishirx34 showed me.

Reply:
Then why did EF do it `Db`?

Reply:
In short, I think we should be consistent with what we already have (`Db`) -- unless we have many places that are also `DB` then I guess it's a toss-up.

Reply:
EF likely got it wrong too...it's a common mistake.

I don't think it's a big deal but `DB` is technically correct. For example, `System.IO` is not `System.Io`. I don't think we should be resistant to using the correct syntax just because we've done it incorrectly in the past. I'm fine with whatever we choose. Xavier did have it as `Db2Catalog` before and I asked him to change it to `DB2Catalog` so I feel bad for having him change it back.

Reply:
Talked offline. Although the Microsoft guidelines on pascalcase seem to imply that `DB` is the right casing, `Db` seems to be used everywhere, in not only EF but also `System.Data.Common`. Given that, it seems like being consistent is a much better choice here.

Reply:
Ok, reverting it all to `Db` lower case :)

Reply:
Fixed in https://github.com/NuGet/NuGet.Services.Metadata/pull/524/commits/33591d3f6ae50b3af5a9d637a7339a5f0db5742b and https://github.com/NuGet/NuGet.Services.Metadata/pull/524/commits/f21017a88db277bb418a977f507bf96f2525af7a